Runtime/SimpleGraphQL/HttpUtils.cs

```diff
@@ -84,6 +87,7 @@ public static void Dispose()
             string payload = request.ToJson();
             
             var requestMessage = new HttpRequestMessage();
+            //byte[] payload = request.ToBytes(serializerSettings);
```

Comment:
@skjalgsm-statespace I would like to ask you a review on this. Basically you and @GioStatespace reworked the same line, and he changed more stuff, so serializer settings could no more needed here.